build.yaml

```diff
@@ -45,7 +45,19 @@ build:
 
       echo "========== Copying ssl files to $HOME/ssl =========="
       cp -r /home/jenkins/ccm/ssl $HOME/ssl
-
+      
+      export SNI_ENABLED="true"
```

Comment:
NIT: given that SNI project uses a single DSE version, we could build it on a single dimension of the matrix, e.g., build it only on `'3.11'` jobs

Reply:
done, already checked in jenkins that it is correctly skipping the cloud tests for targets other than `3.11` do you agree with the way I implemented this in `build.yaml`? It's not very pretty but I didn't know any other way to do this

Reply:
yeah, looks good. We're already overriding `CASSANDRA_VERSION` so makes a code a little more complicated but its OK :)